fr.handbrake.ghb.Plugin.IntelMediaSDK.json

```diff
@@ -0,0 +1,119 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.IntelMediaSDK",
+    "branch": "1",
+    "runtime": "fr.handbrake.ghb",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//3.36",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "modules": [
+        {
+            "name": "intel-gmmlib",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/gmmlib/archive/intel-gmmlib-20.1.1.tar.gz",
+                    "sha256": "821755657cf51f59d8f3f443c99e3ec9f28d897ff65c219c6a119e4acb5a2ac7"
+                }
+            ],
+            "buildsystem": "cmake",
```

Comment:
`cmake-ninja`

Reply:
Why?  This is a cmake build, not ninja/meson

Reply:
ah, i think i see. ninja is more efficient than make?

Reply:
Yes, minor thing, but we use whenever it works.